docs/Cluster.md

```diff
@@ -93,24 +93,38 @@ Generate a self-signed certificate (set a strong password):
 $ keytool -genkeypair -v \
           -keystore akka-cluster-tls.jks \
           -dname "O=ACME, C=FR" \
-          -keypass:env <password> \
-          -storepass:env <password> \
+          -keypass <password> \
+          -storepass <password> \
           -keyalg RSA \
           -keysize 4096 \
           -validity 9999
 ```
 
-Copy the resulting certificate to your `.eclair` directory:
+Copy the resulting certificate to the `.eclair` directory on your backend node and all your frontend nodes:
 ```shell
 $ cp akka-cluster-tls.jks ~/.eclair
 ```
+Add this to `eclair.conf` on all your frontend nodes:
+```
+akka.remote.artery.transport = "tls-tcp"
+```
 
-Add this to your `eclair.conf`:
+Start all your frontend nodes with the following environment variables:
```

Comment:
The part you have added is useful, but unrelated to encryption of akka communication. Needs to be put in a different section.

```suggestion
#### Run cluster nodes on separate servers

Start all your frontend nodes with the following environment variables:
```